src/clients/BridgeClient.ts

```diff
@@ -41,33 +40,6 @@ export class BridgeClient implements IClient {
         return this._commandRunner.outputEmitted;
     }
 
-    public async getVersionAsync(): Promise<string> {
```

Comment:
💡 we can simplify this functions to be used for cases where say there is a breaking change and we need to know what is the current version of staging, vs local vs prod.

I have seen some scenarios where folks use it only for local debugging and I wonder will this ever be an information which we will need.